portality/tasks/helpers/background_helper.py

```diff
@@ -170,9 +185,10 @@ def submit_by_background_job(background_job, execute_fn):
     execute_fn.schedule(args=(background_job.id,), delay=10)
 
 
-def create_execute_fn(redis_huey, task_factory: TaskFactory):
-    @redis_huey.task()
-    @write_required(script=True)
+def create_execute_fn(task_queue: RedisHuey, task_factory: TaskFactory, task_name=None, script=True):
```

Comment:
This didn't seem to work for me, it was adding a task called `queue_task__execute_fn` to the queue.  I've changed this to 

```
@huey_helper.register_execute(is_load_config=False)
def article_bulk_create(job_id):
    job = models.BackgroundJob.pull(job_id)
    task = ArticleBulkCreateBackgroundTask(job)
    BackgroundApi.execute(task)
```

in line with what a lot of the other tasks do, and that's naming the job correctly